docs/source/developer_guide/core/caching.rst

```diff
@@ -3,10 +3,30 @@ Caching: implementation details
 
 This section covers some details of the caching mechanism which are not discussed in the :ref:`user guide <caching>`. If you are developing a plugin and want to modify the caching behavior of your classes, we recommend you read :ref:`this section <caching_matches>` first.
 
+Controlling caching
+-------------------
+
+When developing calculation and data classes, there are some methods you can use to determine how the hash is created:
+
+* To ignore specific attributes, a ``Node`` subclass can have a ``_hash_ignored_attributes`` attribute. This is a list of attribute names which are ignored when creating the hash.
+* For calculations, the ``_hash_ignored_inputs`` attribute lists inputs that should be ignored when creating the hash.
```

Comment:
This may not even be a comment that can be resolved here - but isn't the plural of input, input?

Reply:
not in this case https://english.stackexchange.com/a/180366

Reply:
Great! Thanks :smiley: That saves me and us a lot of changes :sweat_smile: